forc/src/utils/parameters.rs

```diff
@@ -0,0 +1,30 @@
+pub const DEFAULT_BYTE_PRICE: u64 = 0;
+pub const DEFAULT_GAS_LIMIT: u64 = fuel_tx::consts::MAX_GAS_PER_TX;
+pub const DEFAULT_GAS_PRICE: u64 = 0;
```

Comment:
One alternative approach that can be nice for sets of copy data defaults like this might be:
```rust
impl TxParameters {
    pub const DEFAULT: Self = Self {
        byte_price: 0,
        gas_limit: fuel_tx::consts::MAX_GAS_PER_TX,
        gas_price: 0,
    };
}
```
and then const access individual field defaults via
```rust
TxParameters::DEFAULT.gas_price
```
And implement `Default` with:
```rust
impl Default for TxParameters {
    fn default() -> Self {
        Self::DEFAULT
    }
}
```
All that said, I think the existing approach is fine too (and was the only way before associated consts landed anyway).

Reply:
Fixed in [17f308c](https://github.com/FuelLabs/sway/pull/1023/commits/17f308ce215b56ec2dd7ca0570737aab04a5c394)